docs/source/membership/membership.md

```diff
@@ -172,11 +172,11 @@ A local MSP folder contains the following sub-folders:
 
   **Note:** If you are using a [Hardware Security Module(HSM)](../hsm.html) for key management, this folder is empty because the private key is generated by and stored in the HSM.
 
-* **signcert:** For a peer or orderer node (or in a client's local MSP) this folder contains the node's **signing key**. This key matches cryptographically the node's identity included in **Node Identity** folder and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.
+* **signcert:**  For a peer or orderer node (or in a client’s local MSP) this folder contains the node’s certificate issued by CA. This certificate contains the node's **public key**. While private key is used to digitally sign a transaction, public key is used to verify that the signature attached to the transaction is valid.
 
-  This folder is mandatory for local MSPs, and must contain exactly one public key. Obviously, access to this folder must be limited only to the identities of users who have  administrative responsibility on the peer.
+  This folder is mandatory for local MSPs, and must contain exactly one public key. Obviously, access to this folder must be with all the members of a channel.
 
-  Configuration of a **channel MSP** does not include this folder, as channel MSPs solely aim to offer identity validation functionalities and not signing abilities.
+  Configuration of a **channel MSP** includes this folder, as channel MSPs offer identity validation functionalities.
```

Comment:
This is not correct, see how channel MSP's are defined in the channel configuration (ref. configtx.yaml).

Reply:
Checked it. I don't see the signcert folder in channel MSP in the given examples but I wrote this line based on the understanding that since Channel MSP aim to offer identity validation functionalities, public certificates of channel members would be needed. Please correct me if my understanding is incorrect.

Reply:
Channel configuration does not and should not include sign-certs.  Because the configuration for the local MSP and channel MSP shares the same data structure, you may erroneously see a signcert field in the channel configuration, but this should never be populated.

Reply:
Got it. Thanks.

Reply:
Sorry for late response. While re-checking this sentence, I got 2 more doubts:
(1)When above sentence says "...as channel MSPs solely aim to offer identity validation functionalities and not signing abilities.", it seems to imply that if signcert folder had been included, it would have provided signing abilities which seems incorrect. 

(2) How does channel MSP then perform identity validation? Will they extract certificate from the incoming message and do identify validation?